resource-definitions/terraform-driver/runner-pod-configuration/k8s-cluster-refs.tf

```
@@ -0,0 +1,30 @@
+resource "humanitec_resource_definition" "aks_resource_cluster" {
```

```suggestion
resource "humanitec_resource_definition" "aks_aad_resource_cluster" {
```
Align the resource name with the reference in the matching criteria below. Either change this name here to `aks_aad_resource_cluster`, or the reference in the matching criteria to `aks_resource_cluster`.